README.md

```diff
@@ -1,23 +1,7 @@
-[![Ohloh SCM on OpenHub](https://www.openhub.net/p/ohloh_scm/widgets/project_partner_badge.gif)](https://www.openhub.net/p/ohloh_scm) [![Build Status](https://travis-ci.org/blackducksoftware/ohloh_scm.svg?branch=master)](https://travis-ci.org/blackducksoftware/ohloh_scm)
```

Comment:
We should leave in our badges, etc. If they don't look good because we're not keep up with these aspects, we should fix the issue

Reply:
Thanks for pointing this out. I missed adding it.